custom_components/blitzortung/config_flow.py

```diff
@@ -38,20 +48,57 @@ async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Handle the initial step."""
-        if user_input is not None:
-            await self.async_set_unique_id(
-                f"{user_input[CONF_LATITUDE]}-{user_input[CONF_LONGITUDE]}"
-            )
-            self._abort_if_unique_id_configured()
-            return self.async_create_entry(
-                title=user_input[CONF_NAME],
-                data=user_input,
-                options={
-                    CONF_RADIUS: DEFAULT_RADIUS,
-                    CONF_MAX_TRACKED_LIGHTNINGS: DEFAULT_MAX_TRACKED_LIGHTNINGS,
-                    CONF_TIME_WINDOW: DEFAULT_TIME_WINDOW,
-                },
-            )
+        errors = {}
+        if user_input is not None:            # Validate input based on tracking mode
+            if user_input[CONF_TRACKING_MODE] == "Static":
```

Comment:
In the `user` step there should be a selection of the operating mode (static or device tracker) and depending on the result a step for configuring the coordinates or for selecting the device tracker entity should be displayed.